src/libertem/api.py

```diff
@@ -1510,3 +1512,53 @@ def _exit():
             weak_ctx().close()
 
         atexit.register(_exit)
+
+    def convert_dataset(
+        self,
+        dataset: DataSet,
+        path: os.PathLike,
+        progress: bool = False,
```

Comment:
Discussion: `path`, `progress` could be keyword-only arguments

Reply:
+1